task_12/index.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
```

Comment:
HTML 5 does not require `/` for self closing tags. By the way, it's recommended to avoid such syntax in HTML 5.

Reply:
Thank you, did not know